### PR TITLE
test(forms): add failing test for dependsOn

### PR DESCRIPTION
**Pull request by yusufsafak**

### CODE REVIEW

File: packages/cerebral-forms/src/utils/validate.test.js [hunk 1]
```diff
@@ -260,6 +260,47 @@ describe('validate', () => {
...
+          ]
+        }
+      })
+      controller.getSignal('passwordFieldChanged')({value1: 'password'})
+      assert.equal(controller.getState('form.password.isValid'), false)
+      assert.equal(controller.getState('form.confirmPassword.isValid'), false)
+      controller.getSignal('confirmPasswordFieldChanged')({value2: 'password'})
+      assert.equal(controller.getState('form.password.errorMessage'), null)
+      assert.equal(controller.getState('form.confirmPassword.errorMessage'), null)
+      assert.equal(controller.getState('form.password.isValid'), true)
```
Comment (henri-hulski) [inline packages/cerebral-forms/src/utils/validate.test.js]: Adding `dependsOn: 'form.password'` to `confirmPassword` fixes that,  but not sure if this should be necessary.

**Comment by edgesoft** [comment]
@henri-hulski Is this still valid?

**Comment by henri-hulski** [comment]
@edgesoft I think nothing changed here.

**Comment by christianalfoni** [comment]
Depends on will not be needed in next version of the "derived" forms, so suggest we close this one for now

**Pull request closed without merging**